sebastien15/capstoneproject
# Ft save query/question

#### What does this PR do?

> This PR save queries/question to mongodb

#### Description of Task to be completed

> i added route and method to save query to mongodb

#### How should this be manually tested?

- Clone this repo with `git clone git@github.com:sebastien15/myportifolio.git` 
- Checkout to `branch_name`
- npm install nodemon -g
- npm install
- npm start
- run localhost:3000/saveContact in post man make sure it is a post method
- add the firstName,secondName,email and message fields in the postman and send i


#### Any background context you want to provide?
>N/A
#### What are the relevant Trello story?
>https://trello.com/c/T3UGgcUh
#### Screenshots (if appropriate)
>
![Uploading save contact.PNG…]()
